tools/parabricks/parabricks_fq2bam.xml

```diff
@@ -0,0 +1,123 @@
+    </requirements>
+    <command detect_errors="aggressive">
+    <![CDATA[
+        #if $reference_source.reference_source_selector == 'cached':
+            ln -f -s '$reference_source.ref_file.fields.path' reference.fasta && 
+        #end if
+        #set $o_mappings = 'output_mapped_file.bam'
+        ln -f -s '$fastq_input.fastq_input1' input1.fastq &&
+        #if $fastq_input.fastq_input_selector == 'paired':
+            ln -f -s '$fastq_input.fastq_input2' input2.fastq &&
+            pbrun fq2bam --low-memory --ref reference.fasta --in-fq input1.fastq input2.fastq --out-bam '${o_mappings}' &&
+        #else:
+            pbrun fq2bam --low-memory --ref reference.fasta --in-se-fq input1.fastq --out-bam '${o_mappings}' &&
+        #end if  
+        cp '${o_mappings}' '$output_mapping'
+    ]]>
+    </command>
+    <inputs>
+        <conditional name="reference_source">
+            <param name="reference_source_selector" type="select" label="Will you select a reference genome from your history or use a built-in index?" help="Built-ins were indexed using default options. If you would like to perform self-mapping select `history` here, then choose your input file as reference (currently not supported).">
+                <option value="cached">Use a built-in genome index</option>
+            </param>
+            <when value="cached">
+                <param name="ref_file" type="select" label="Using reference genome" help="Select genome from the list">
+                    <options from_data_table="all_fasta">
+                        <filter type="sort_by" column="2" />
+                        <validator type="no_options" message="No reference genomes are available" />
+                    </options>
+                    <validator type="no_options" message="A built-in reference genome is not available for the build associated with the selected input file"/>
+                </param>
+            </when>
+        </conditional>
+        <conditional name="fastq_input">
+            <param name="fastq_input_selector" type="select" label="Single or Paired-end reads" help="Select between paired and single end data">
+                <option value="single">Single</option>
+                <option value="paired">Paired</option>
+            </param>
+            <when value="single">
+                <param name="fastq_input1" type="data" format="fastqsanger,fasta" label="Select fastq dataset" help="Specify dataset with single reads"/>
+            </when>
+            <when value="paired">
+                <param name="fastq_input1" type="data" format="fastqsanger,fastq" label="Select first set of reads" help="Specify dataset with forward reads"/>
+                <param name="fastq_input2" type="data" format="fastqsanger,fastq" label="Select second set of reads" help="Specify dataset with reverse reads"/>
+            </when>
+        </conditional>
```

Comment:
https://github.com/galaxyproject/tools-iuc/blob/bbf307ebb324e2b3278338b2b15672f071c98edc/tools/bwa_mem2/bwa-mem2.xml#L131-L135

single, paired and paired collection is what should be supported (and paired collection is more important to have than paired). It's ok to just steal code from bwa-mem/bwa-mem2 :-)